internal/controller/apiextensions/definition/reconciler.go

```diff
@@ -585,48 +585,41 @@ func (r *Reconciler) CompositeReconcilerOptions(ctx context.Context, d *v1.Compo
 
```

Comment:
This file is worth a closer review, to make sure I didn't mess up any of the feature flag handling logic.